graphql2/graphqlapp/alert.go

```diff
@@ -375,6 +375,28 @@ func (a *Alert) RecentEvents(ctx context.Context, obj *alert.Alert, opts *graphq
 	return conn, err
 }
 
+// PendingNotifications returns a list of notifications that are waiting to be sent for some reason such as throttling
+func (a *Alert) PendingNotifications(ctx context.Context, obj *alert.Alert) ([]graphql2.AlertPendingNotification, error) {
+	var result []graphql2.AlertPendingNotification
+
+	if obj.Status != alert.StatusTriggered {
+		return result, nil
+	}
+
+	rows, err := a.AlertStore.FindPendingNotifications(ctx, obj.ID, obj.ServiceID)
```

Comment:
Name this something other than rows, it implies it's a `sql.Rows` which would need to be closed. something like `pending` or `notifications` or just `p` since it's only used for a few lines that are close by without scrolling